lesson2/homework_argirovatina.rb

```diff
@@ -0,0 +1,25 @@
+module M
+
+  def self.included(base)
+    base.extend(C)
+    p base.custom_class_method
+    p base.new.custom_instance_method
+  end
+
+  def custom_instance_method
+     Foo.instance_methods - Object.instance_methods
```

Comment:
It's closer, but still not exactly correct)
The problem is that your methods calculate and provide information about added methods, in other words, about themselves. Just forget about them, **leave their body completely empty**.
Solution is really simple: just move this lines:
```
Foo.instance_methods - Object.instance_methods
Foo.methods - Object.methods
```
into `#included` hook with appropriate explanations, like `class methods: ...`, `instance methods: ...`